src/components/supportWithdraw/EthereumToMarketToken.vue

```diff
@@ -57,13 +58,11 @@ export default class EthereumToMarketToken extends Vue {
   public get marketTokenValueInEth(): number {
```

Comment:
Might be worth a comment to clarify what this returns..whole ETH per whole CMT right? (vs something else like ETH in wei per whole CMT)